spark/agent/src/test/scala/za/co/absa/spline/harvester/dispatcher/HttpLineageDispatcherSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester.dispatcher
+
+import org.mockito.Mockito._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatestplus.mockito.MockitoSugar
+import scalaj.http._
+import za.co.absa.spline.harvester.exception.SplineNotInitializedException
+
+class HttpLineageDispatcherSpec extends AnyFlatSpec with MockitoSugar {
+
+  behavior of "HttpLineageDispatcher"
+
+  val dummyUrl = "http://dummyUrl"
+
+  val httpMock = mock[BaseHttp]
```

Comment:
Minor: Even though it's a unit-test these val's had better to be marked as `private`. At least because it removes those IDE's warnings (otherwise it complains about missing type annotation on public members)